docs/getting_started/why_use_GTFS.md

```diff
@@ -1,42 +1,104 @@
+    [:octicons-arrow-right-24: Learn more](../features/overview)
 
-## Open Data means more opportunity and choices
+</div>
 
-GTFS is an Open Standard. This means that agencies can make information available using any of the tools which already support GTFS (including simple text editing or a spreadsheet). Open standards lead to the creation of data that can be easily shared. The feeds can be used by trip planners such as Google, Apple, Transit App, Open Trip Planner, and even apps created by riders. Anyone who wants to provide accurate and up-to-date transit information can use GTFS feed to do so.
+## Improved Rider Experience
 
-Some riders like to use different apps depending on their needs—having GTFS lets riders choose what trip planning app suits them best. Some apps may be more accessible or better at providing information for riders with disabilities, some may be simpler and easier to use, and sometimes riders just want the newest app.
+GTFS empowers riders with accurate schedules, real-time updates, and a wider choice of apps, leading to reduced wait times and informed travel decisions. Here's how GTFS enhances the rider experience:
 
-## Maintained by the community
+- **Up-to-Date Information:** GTFS provides riders with access to real-time schedules, routes, and stop information, facilitating informed travel decisions.
+- **Reduced Wait Times:** Real-time information (GTFS Realtime) offers live updates on vehicle locations, reducing wait times and enhancing the overall travel experience.
+- **Wider App Choice:** Access to GTFS data enables riders to choose from a variety of transit apps, ensuring they find the app that best suits their needs and preferences.
 
-As an open data standard, GTFS is driven by an open community where anyone can propose changes or improvements. The specification evolution is overseen by an independent nonprofit organization called [MobilityData](https://mobilitydata.org/), and its changes are guided by the following principles:
+<div class="grid cards" markdown>
 
-- Feeds should be easy to create and edit
-- Feeds should be easy to parse
-- The specification is primarily concerned with passenger information
-- Changes to the specification should be backwards-compatible
-- Speculative features are discouraged
+-   :material-bus-alert:{ .lg .middle } __Benefits of Realtime Information (RTI)__
 
-The Reference documents for [GTFS Schedule](/documentation/schedule/reference/) and [GTFS Realtime](/documentation/Realtime/reference/) are officially hosted in a [GitHub repository](https://github.com/google/transit), where changes are proposed and reviewed by the community.
 
+    [:octicons-arrow-right-24: Learn more](https://realtimetransit.info/)
```

Comment:
I'm not sure we want to link to this page, it has good information but it's a bit strange that it just exists on it's own and looks a bit outdated.

Reply:
I don't think the information is outdated and is relevant. But, there is a case to be made if this is the best place to put it. It would answer the question of Why GTFS / Realtime though.

Reply:
+1 on notlinking to this site that isn't maintained, and it gives the impression that GTFS Realtime lives on a different site.

I think eventually we want to bring the info that is uniquely appearing on this site here if any (out of scope for this iteration) and archive it. 

IMO this page answers "Why GTFS Realtime" very well.